src/devices/cellular.c

```diff
@@ -66,57 +66,6 @@ static const struct at_config* get_safe_at_config()
         return &cfg;
 }
 
-static void cell_serial_tx_cb(const char *data)
-{
```

Comment:
This is being removed because it is no longer usable code now that we handle printing all serial messages from within the serial code.  This was here because it was the code that used to get invoked when I needed to print only the messages for the cellular device.  This patch is a much improved version of what this was.